src/React.re

```diff
@@ -390,3 +390,10 @@ external useImperativeHandle7:
 
 [@bs.set]
 external setDisplayName: (component('props), string) => unit = "displayName";
+
+[@bs.get]
+external getName: component('props) => Js.undefined(string) = "name";
+
+[@bs.get]
+external getDisplayName: component('props) => Js.undefined(string) =
+  "displayName";
```

Comment:
Let's use [@bs.return nullable] on these - see #526 as an example to get a better api on the Reason side. Also I think we should make the API match and just call displayName and name.

Reply:
Thanks @rickyvetter 🙌 

`[@bs.return nullable]` is nice!